File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;
class Main{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        
        int arr[] = new int[9];
        int max = arr[0];
        int indext=0;
        for(int i=0; i<arr.length; i++){
            arr[i] = sc.nextInt();
            if(max<arr[i]){
                max = arr[i];
                indext = i + 1;
            }
        }
        System.out.println(max);
        System.out.println(indext);
    }
}

File: 백준/Bronze/2562. 최댓값/최댓값.c
#include <stdio.h>
main(){
    int arr[9]; 
    for(int i=0; i<9; i++){
         scanf("%d", &arr[i]);      
    }
    int max = arr[0];
    int indext=1;
    for(int j=0; j<9; j++){
        if(max<arr[j]){
            max = arr[j];
            indext = j+1;
        }      
    }
     printf("%d\n", max);
     printf("%d", indext);
}